dev/create-release/release-build.sh

```diff
@@ -345,16 +345,14 @@ if [[ "$1" == "publish-snapshot" ]]; then
   #  -DskipTests $SCALA_2_12_PROFILES $PUBLISH_PROFILES clean deploy
 
   # Clean-up Zinc nailgun process
-  /usr/sbin/lsof -P |grep $ZINC_PORT | grep LISTEN | awk '{ print $2; }' | xargs kill
+  lsof -P |grep $ZINC_PORT | grep LISTEN | awk '{ print $2; }' | xargs kill
```

Comment:
Hm, @holdenk, I thought the full path of `lsof` is required for non-root user in few OSs.

Reply:
Huh that's weird, not something I expected. I think in that case the user could just add `/usr/sbin` to their `$PATH` (whereas on some systems lsof is in `/usr/bin` not `/usr/sbin` so fixing that would require root).

Reply:
If this is a thing people are likely to run into I can add some code to check for lsof in different places under usr, but that seems brittle. Checking on one of the jenkins machines, my local laptop, and a random debian server I run `lsof` all works as a non-root user just fine. (Although searching on Google does indeed result in some folks who have had to either add `/usr/sbin` or other dir to their path as a non-root user).